### PR TITLE
[MOB-4565] Update README.md

### PR DESCRIPTION
**Pull request by AliAbdelfattah**
## Description of the change
- Revisit README format.
- Document the unsupported features.
- Remove irrelevant info from the integration steps.
- Add the compileSdkVersion requirement.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
### Development
- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
### Code review 
- [X] This pull request has a descriptive title and information useful to a reviewer
- [X] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
 
 For more info, visit [Instabug.com](https://instabug.com).
 
+### Prerequisites
+
+- Starting from Instabug-Cordova v9, we require the `compileSdkVersion` on Android to be set to a minimum of `29`.
```
Comment (bsameh-at-instabug) [inline README.md:11]: Can we mention which file needs to be changed + add a snippet showing where the property is in that file?

Comment (AliAbdelfattah) [reply README.md]: Added 👍

File: README.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
...
 For more info, visit [Instabug.com](https://instabug.com).
 
+### Prerequisites
+
+- Starting from Instabug-Cordova v9, we require the `compileSdkVersion` on Android to be set to a minimum of `29`. It can be set inside the app's `build.gradle` file as below:  
+
+	```
+	    android {
+	        compileSdkVersion 29
+	        ...
```
Comment (AliAbdelfattah) [inline README.md:16]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -85,13 +73,17 @@ cordova.plugins.instabug.activate(
 
 You can change the invocation event with any of the following: `'button'`, `'screenshot'`, `'swipe'`, or `'shake'`.
 
-##### Ionic
+⚠️ TypeScript users, make sure you declare cordova at the beginning of your App TS class:
 
-If you're using this plugin with Ionic, you need to add this line at the beginning of your **app.component.ts** class.
```
Comment (bsameh-at-instabug) [inline README.md:90]: Do you think removing this is necessary? I thought using the exact file name was useful.

Comment (AliAbdelfattah) [reply README.md:90]: The file doesn't necessarily have to be named like that. Also, our integration doesn't have to be placed in that file.
Actually, this tip is general knowledge for TS users, if they're using Cordova, they most probably already know it (we can actually remove the whole tip :D)

File: README.md [hunk 4]
```diff
@@ -85,13 +73,17 @@ cordova.plugins.instabug.activate(
 
 You can change the invocation event with any of the following: `'button'`, `'screenshot'`, `'swipe'`, or `'shake'`.
 
-##### Ionic
+⚠️ TypeScript users, make sure you declare cordova at the beginning of your App TS class:
```
Comment (AliAbdelfattah) [inline README.md:76]:
```suggestion
⚠️  TypeScript users, make sure you declare `cordova` at the beginning of your app class (app.component.ts):
```

File: README.md [hunk 5]
```diff
@@ -85,13 +73,17 @@ cordova.plugins.instabug.activate(
...
-```
-declare let cordova: any;
-```
+## Features Not Yet Supported
+- User steps.
+- Repro steps.
+- Network logging.
+- View hierarchy.
+- Crash reporting.
+- Push notification for in-app messaging.
```
Comment (AliAbdelfattah) [inline README.md:86]: Should we push this to the README? @bsameh-at-instabug

**Review by bsameh-at-instabug — APPROVED** [review]

**Pull request merged**